src/test/java/com/slack/kaldb/logstore/LuceneIndexStoreImplTest.java

```diff
@@ -61,7 +61,6 @@ public void testSimpleIndexAndQuery() {
           findAllMessages(
               forgivingLogStore.logSearcher, MessageUtil.TEST_INDEX_NAME, "Message1", 10, 1);
       assertThat(results.size()).isEqualTo(1);
-      assertThat(forgivingLogStore.metricsRegistry.getMeters().size()).isEqualTo(4);
```

Comment:
every time we add a new metric we'll have to change all of this. I don't think this is adding any value.

If the test cares about the metric it should just assert that like it's doing here. So we can remove the total registry count

Reply:
Yes, they have outlived their usefulness. I have been removing these also. Please feel free to remove them when you find them.